image/scanner/konflux/Dockerfile

```
@@ -0,0 +1,77 @@
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8-minimal
+ARG BASE_TAG=latest
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+ENV CGO_ENABLED=1
+ENV GOFLAGS=""
+ENV CI=1
+
+COPY . /src
+WORKDIR /src
+
+RUN scripts/konflux/fail-build-if-git-is-dirty.sh
+
+RUN unzip -j blob-repo2cpe.zip -d image/scanner/dump/repo2cpe && \
+    unzip -j blob-k8s-definitions.zip -d image/scanner/dump/k8s_definitions && \
+    unzip -j blob-nvd-definitions.zip -d image/scanner/dump/nvd_definitions
+
+RUN echo "version: $(scripts/konflux/version.sh)" && \
```

It could be
```suggestion
RUN echo -n "version: " && scripts/konflux/version.sh && \
```
but really up to you. I just being pedantic.